src/module_ncio.f90

```diff
@@ -1,81 +1,58 @@
 !> @file
-!! Module for reading/writing netcdf files such as the lat/lon grid history files output by the GFS.
-!! writing requires a template file.
+!! @brief Module for reading/writing netcdf files such as the lat/lon grid history files output by the GFS..
+!! @author jeff whitaker <jeffrey.s.whitaker@noaa.gov> @date 201910
+
+!> writing requires a template file.
```

Comment:
The first sentence is the brief description of the module, so should summarize it. Something like: "Module to handle netCDF I/O for UFS."